app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -29,47 +29,73 @@
                     sort=this.sort
                 ) as |SortButton|}}
                     <tr>
-                        <th data-test-header-name rowspan='2'>
-                            <strong>{{t 'institutions.dashboard.users_list.name'}}</strong>
-                            <SortButton @sortBy='user_name' />
+                        <th data-test-header-name rowspan='3'>
+                            <div class="header-content">
```

Comment:
Seems like this `class=` statement is not adding any styling. See my comment on local-class earlier